2023/12/06 16:02:52 [error] 160#160: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.36.1, server: nginxproxymanager, request: "GET /api/ HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "192.168.36.4:81", referrer: "http://192.168.36.4:81/nginx/proxy"
2023/12/06 16:09:58 [error] 239#239: *41 upstream prematurely closed connection while reading response header from upstream, client: 192.168.36.1, server: nginxproxymanager, request: "GET /api/nginx/certificates/test-http?domains=%5B%22ztasecurity.duckdns.org%22%2C%22*.ztasecurity.duckdns.org%22%5D HTTP/1.1", upstream: "http://127.0.0.1:3000/nginx/certificates/test-http?domains=%5B%22ztasecurity.duckdns.org%22%2C%22*.ztasecurity.duckdns.org%22%5D", host: "192.168.36.4:81", referrer: "http://192.168.36.4:81/nginx/certificates"
